### ISSUE TITLE
Providing a configuration for grade-specs does not work

### ISSUE THREAD

**Issue by rjglasse** [slarse/repobee-csvgrades#21]
Whilst passing a grade spec as a command line option works as expected, when moving that grade spec to a configuration file as follows:

`grade_specs = 1:P:[Pp]ass|[Gg]odkän[td]`

the following trace is generated:

```
% repobee grades record -a task-2 --traceback 
[ERROR] invalid format string: 1
[ERROR] Critical exception
Traceback (most recent call last):
  File "/Users/ric/.repobee/env/lib/python3.8/site-packages/_repobee/main.py", line 170, in _main
    _repobee.cli.dispatch.dispatch_command(
  File "/Users/ric/.repobee/env/lib/python3.8/site-packages/_repobee/cli/dispatch.py", line 47, in dispatch_command
    else ext_cmd.command()
  File "/Users/ric/.repobee/env/lib/python3.8/site-packages/repobee_csvgrades/csvgrades.py", line 87, in command
    callback(self.args)
  File "/Users/ric/.repobee/env/lib/python3.8/site-packages/repobee_csvgrades/csvgrades.py", line 59, in callback
    grade_specs = list(
  File "/Users/ric/.repobee/env/lib/python3.8/site-packages/repobee_csvgrades/_containers.py", line 47, in from_format
    raise plug.PlugError(
repobee_plug.exceptions.PlugError: invalid format string: 1
```

I've tried a combination of different quotation marks, but do not think that should matter.

**Comment by slarse** [comment]
This is related to #20

**Comment by slarse** [comment]
When repobee/repobee#745 is fixed, both this issue and #20 amount to updating the documentation. The grade specs will be configurable like so:

```
[csvgrades]
grade_specs = 1:P:[Pp]ass 2:F:[Ff]ail 3:K:[Kk]omplettering
```
